src/components/forms/bouquet/BouquetOwnerForm.vue

```diff
@@ -23,8 +23,25 @@ const choice: Ref<'organization' | 'owner'> = ref(
   topic.value.organization != null ? 'organization' : 'owner'
 )
 
-const selectedAnyOrganization: Ref<Organization | undefined> = ref(undefined)
-const selectedOwnOrganization: Ref<string | undefined> = ref(undefined)
+const organizations = computed(() => userStore.data?.organizations || [])
+
+// checks if current owner is a user's org and return its id
+const ownOrganizationIndex = computed(() => {
+  return organizations.value.findIndex(
+    (org) => org.id === topic.value.organization?.id
+  )
+})
+// both models check the above index to determine the default owner to display
+const selectedAnyOrganization: Ref<Organization | undefined> = ref(
+  ownOrganizationIndex.value < 0 && !topic.value.owner
+    ? topic.value.organization
+    : undefined
+)
+const selectedOwnOrganization: Ref<number | string | undefined> = ref(
```

Comment:
```suggestion
const selectedOwnOrganization: Ref<number | undefined> = ref(
```

Je ne crois pas qu'on puisse avoir une string ici.

Reply:
On a une string quand on sélectionne une option côté client. Vu qu'on utilise le DsfrSelect, j'ai pas pu forcer un number.
Donc dans le code on ne passe jamais de string mais en réalité le v-model récupère la value de l'option en string.
C'est peut-être pas grave du coup mais j'ai préféré pas prendre le risque 😬